Automattic/pocket-casts-android
# Bump Spotless and ktLint versions

## Description

While working on migrating [ShiftyCustomAudio.java](https://github.com/Automattic/pocket-casts-android/blob/61fe9bf02411dbb158ec838d2fc4834099e99a4b/modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/ShiftyCustomAudio.java) from Java to Kotlin it was convenient to use `rangUntil()` (`..<`) operator. However, Spotless started crashing after using it since we used too old of a version.

## Testing Instructions

Review my manual changes to the code that Spotless couldn't auto-fix on its own.

1d3b337cb5bcd98212aab1b8294234c6a6ab7de6 - Version bump.
9823709e26e663551a6ee6905821bf462536d451 - Add suppression when generating `ThemeStyle.kt`. It doesn't make sense to require some specific enum naming when code is auto-generated.
38f65a64af47958aad293f2cbcb5bf9a9c58ddb7 - Fix code and comment inlining. I decided to remove the `/* synthetic accessor */` comment from properties instead of putting it above them. I think they aren't necessary and Lint should already [inform](https://android.googlesource.com/platform/tools/base/+/studio-master-dev/lint/libs/lint-checks/src/main/java/com/android/tools/lint/checks/SyntheticAccessorDetector.kt) about it.
ef4e169380d3ba3bdaf583ebfdd6f73acd62905e - Align file/class naming.


You can also check e06d3382f91d1840ce94f70133ad17faf3bf0845 for changes that Spotless applied during an auto-fix. If you don't like some changes we can probably reconfigure Spotless to what we like. Key changes are:
* Some import ordering.
* A required comma after last argument.
* Empty spaces between annotated properties.

## Screenshots or Screencast 


## Checklist
 
#### I have tested any UI changes...
